dart/lib/src/sentry_options.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:http/http.dart';
+import 'package:meta/meta.dart';
+
+import 'protocol.dart';
+import 'utils.dart';
+
+/// Sentry SDK options
+class SentryOptions {
+  /// Default Log level if not specified Default is DEBUG
+  static final SeverityLevel defaultDiagnosticLevel = SeverityLevel.debug;
+
+  /// The DSN tells the SDK where to send the events to. If this value is not provided, the SDK will
+  ///  just not send any events.
+  final String dsn;
```

Comment:
if we decide to use the callback: https://github.com/getsentry/sentry-dart/pull/108/files#r503897735

we can't make them `final`

Reply:
yes indeed ! the class is now mutable.